cog/tests/test_daily_charge.py

```
@@ -0,0 +1,161 @@
+
+        for last_charge, executed_at in test_cases_true:
+            with self.subTest(last_charge=last_charge, executed_at=executed_at):
+                self.assertTrue(
+                    Charge.is_cross_day(
+                        datetime.strptime(last_charge, "%Y-%m-%d %H:%M:%S"),
+                        datetime.strptime(executed_at, "%Y-%m-%d %H:%M:%S"),
+                    )
+                )
+        for last_charge, executed_at in test_cases_false:
+            with self.subTest(last_charge=last_charge, now=executed_at):
+                self.assertFalse(
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertFalse`

Replace `assertFalse(...)` with `assert ...`

(PT009)